sdk/src/test/java/com/hcaptcha/sdk/HCaptchaJSInterfaceTest.java

```diff
@@ -69,6 +69,7 @@ public void full_config_serialization() throws JsonProcessingException, JSONExce
                 .reportapi(reportapi)
                 .host(host)
                 .resetOnTimeout(true)
+                .showDialog(false)
```

Comment:
Should we keep the existing tests when `showDialog=true` and add new tests for `showDialog=false`?